qiskit_ibm_experiment/client/experiment.py

```diff
@@ -179,7 +182,9 @@ def experiment_plot_update(
         Returns:
             JSON response.
         """
-        return self.api.update_plot(experiment_id, plot, plot_name)
+        return self.api.update_plot(
+            experiment_id, plot, plot_name
+        )
```

Comment:
Do you want to change the type of return value also here, for consistency between the upload and update methods?

Reply:
Not right now - this was in the original code and this PR won't change it.